frontend/src/components/SystemSubmitDrawer/index.tsx

```diff
@@ -453,7 +479,11 @@ export function SystemSubmitDrawer(props: Props) {
                 label="Output Lang"
                 rules={[{ required: true }]}
               >
-                <Input />
+                <Select
```

Comment:
I think it may be a better user experience if we show the language name instead of the language code when they are selected (or show both). You may already know this but I think the easiest way is to use `{label: name, value: code}` for options ([doc](https://ant.design/components/select/)).